### PR TITLE
Merge [WEB-976] Add Uploader Banner (v1.37.1 rc.1) to Master

### PR DESCRIPTION
**Pull request by thacherT1D**
Jira Ticket: https://tidepool.atlassian.net/browse/WEB-976

- Created banner for minimum uploader version
- Small style update to banners to center close icon
- Added tests and minor-refactor of banner tests

### CODE REVIEW

File: app/pages/app/app.js [hunk 1]
```diff
@@ -395,6 +416,31 @@ export class AppComponent extends React.Component {
     return null;
   }
 
+  renderUploaderBanner() {
+    this.props.context.log('Rendering update type banner');
```
Comment (cbwebdevelopment) [inline app/pages/app/app.js:420]:
```suggestion
    this.props.context.log('Rendering uploader banner');
```

Comment (thacherT1D) [reply app/pages/app/app.js]: 👍

**Comment by cbwebdevelopment** [comment]
@thacherT1D I've converted this back to Draft state, as it seems you've merged the `release-1.38.0` work into this, and as such, it can't be merged to `master` without everything in the `1.38.0` branch being QA'd.

I'm assuming this wasn't intentional, so I figured it would be best to pull it back to Draft state to make sure we don't accidentally deploy pre-tested code.

**Comment by thacherT1D** [comment]
@cbwebdevelopment you're correct -- thanks for converting it back, I'm taking care of it now

**Pull request merged**